docs/2-techdoc/62304/software-development-maintenance-plan.md

```diff
@@ -87,9 +87,9 @@ as software safety class A. The (external) risk control measures include:
 
 <!-- > List the languages you’ll be using, including compiler and language versions. -->
 
-| System                          | Name   | Version |
-| ------------------------------- | ------ | ------- |
-| Local DICOM server and ML model | Python | 3.9     |
+| System                          | Name   | Version            |
+| ------------------------------- | ------ | ------------------ |
+| Local DICOM server and ML model | Python | {{python_version}} |
```

Comment:
ah... and here it is propagated.
OK.  I'm good with that.

Reply:
:)